manifest/plugin/csi-bizflycloud-controllerplugin.yaml

```diff
@@ -41,6 +41,7 @@ spec:
             - "--csi-address=/var/lib/csi/sockets/pluginproxy/csi.sock"
             - "--timeout=3m"
             - "--feature-gates=Topology=true"
+            - "--extra-create-metadata=True"
```

Comment:
Seem you're mistaken with python syntax here ;)) please use "true" instead of "True"